build/__snapshots__/imported-features.json

```diff
@@ -89,7 +89,6 @@
 	"locked-issue",
 	"mark-merge-commits-in-list",
 	"mark-private-orgs",
-	"mobile-tabs",
```

Comment:
This feature can no longer be disabled now. The delayed addition of the `rgh-mobile-tabs` attribute sometimes caused the overflow mechanism to run before the size change, leaving just 2 tabs visible.